lib/honeybadger/http_adapter/req.ex

```diff
@@ -0,0 +1,49 @@
+if Code.ensure_loaded?(Req) do
+  defmodule Honeybadger.HTTPAdapter.Req do
+    @moduledoc """
+    HTTP adapter module for making http requests with Req.
+
+    You can also override the Req options by updating the configuration:
+
+        http_adapter: {Honeybadger.HTTPAdapter.Req, [...]}
+
+    See `Honeybadger.HTTPAdapter` for more.
+    """
+    alias Honeybadger.{HTTPAdapter, HTTPAdapter.HTTPResponse}
+
+    @behaviour HTTPAdapter
+
+    @impl HTTPAdapter
+    def request(method, url, body, headers, req_opts \\ nil) do
+      headers = headers ++ [HTTPAdapter.user_agent_header()]
+      opts = req_opts || []
```

Comment:
Can we make an empty list the default?

```suggestion
    def request(method, url, body, headers, req_opts \\ []) do
      headers = headers ++ [HTTPAdapter.user_agent_header()]
```